src/app/ApplicationTemplate.Business/ForcedUpdates/IUpdateRequiredService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace ApplicationTemplate.Business;
+
+/// <summary>
+/// This service checks if the application is running the minimum required version.
+/// </summary>
+public interface IUpdateRequiredService : IDisposable
+{
+	/// <summary>
+	/// Event that is raised when the application needs to be updated.
+	/// </summary>
+	event EventHandler UpdateRequired;
```

Comment:
Here you should put a remark to explain the type of the event. Basically to explain that there are no parameters because the update requires the app to be relaunched and therefore the event is like a one-time-deal and it cannot go the other way (it can't say "oh, the update is no longer required")